controller/users.go

```diff
@@ -104,6 +106,21 @@ func (c *UsersController) Create(ctx *app.CreateUsersContext) error {
 		return jsonapi.JSONErrorResponse(ctx, errors.NewUnauthorizedError("account not authorized to create users."))
 	}
 
+	// ----- Ignore users created for Preview environment
+	// TODO remove this when we start using our regular user registration flow in staging environment
+	preview, err := c.checkPreviewUser(ctx.Payload.Data.Attributes.Email)
+	if err != nil {
+		log.Error(ctx, map[string]interface{}{"err": err, "email": ctx.Payload.Data.Attributes.Email}, "unable to parse user's email")
+		return jsonapi.JSONErrorResponse(ctx, errors.NewInternalError(ctx, err))
+	}
+	if preview {
+		log.Info(ctx, map[string]interface{}{"email": ctx.Payload.Data.Attributes.Email}, "ignoring preview user")
+		user := &account.User{Email: ctx.Payload.Data.Attributes.Email, Cluster: ctx.Payload.Data.Attributes.Cluster}
+		identity := &account.Identity{Username: ctx.Payload.Data.Attributes.Username, ProviderType: account.KeycloakIDP}
+		return ctx.OK(ConvertToAppUser(ctx.RequestData, user, identity, true))
```

Comment:
Good! exactly what I was thinking, we don't need to return alarms to the registration app.